mockgcp/mockcompute/networksv1.go

```diff
@@ -207,6 +213,15 @@ func (s *MockService) parseNetworkName(name string) (*networkName, error) {
 	return nil, status.Errorf(codes.InvalidArgument, "name %q is not valid", name)
 }
 
+// parseNetworkSelfLink parses a selfLink string into a networkName.
+// The expected form is `https://www.googleapis.com/compute/beta/projects/*/global/networks/*`.
+func (s *MockService) parseNetworkSelfLink(selfLink string) (*networkName, error) {
+	name := selfLink
+	name = strings.TrimPrefix(name, "https://www.googleapis.com/compute/beta/")
```

Comment:
Maybe change `beta` to `v1`? Our mock uses `v1, I think that's the reason why we see so many 400 Bad Request in http logs, and  error message "network \"https://www.googleapis.com/compute/v1/projects/example-project-01/global/networks/${networkID}\" is not valid"

Reply:
Good call - thanks.  I made it tolerate both beta and v1.  We do now have buildComputeSelfLink, so we know whether it should be beta or v1, but I figure it's better just to accept both (real gcp might do the same thing, I don't know)